src/orb.yml.hbs

```diff
@@ -619,7 +628,7 @@ commands:
       - run:
           name: Retrieve previous task definition and prepare new task definition values
           command: |
-            PREVIOUS_TASK_DEFINITION=$(aws ecs describe-task-definition --task-definition << parameters.family >> --include TAGS)
+            PREVIOUS_TASK_DEFINITION=$(aws ecs describe-task-definition --task-definition << parameters.family >> --profile << parameters.profile-name >> --include TAGS)
```

Comment:
I don't think this will work as expected as the profile-name parameter should also be defined in the command and the job should pass the parameter to the command. Additionally, I would like it if --profile were not set if the value of profile-name is "".